Sources/HTTP/Request/Request.swift

```diff
@@ -1,14 +1,26 @@
 import Foundation
 
+/// HTTP request.
 public final class Request: Message {
```

Comment:
Some examples of reading request would be nice. Like Response they're two critical types that are used often and are best described thoroughly. Unlike the serialisers which you'll only touch if you know what you're doing.